src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080"method="post">
+      <fieldset class="block">
+        <legend class="nameblock">Personal information</legend>
+        <label class="stringdate">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required>
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```

закрывающий тег нужно переносить